### PR TITLE
Fixed title text “Share your experience” is shown in two lines in the tutor registration block.

### PR DESCRIPTION
**Pull request by TSlashDreamy**
I'm not sure if i am even allowed to change any project styles, but i had 3 different options to resolve this problem:

1. Redusing title font-size
2. Redusing padding of InfoCards
3. Removing letter-spacing from title

So, i picked most neutral option and just removed letter-spacing from titles in InfoCards.

Before:
зображення

After:
зображення

### CODE REVIEW

**Review by MickhaelSh — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by tsivkadi — APPROVED** [review]

File: src/components/info-card/InfoCard.styles.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ export const styles = {
     },
     title: {
       typography: { md: 'h4', xs: 'h5' },
+      letterSpacing: '0',
```
Comment (maarekua) [inline src/components/info-card/InfoCard.styles.js:31]: Is this letter spacing according to Figma design?

Comment (TSlashDreamy) [reply src/components/info-card/InfoCard.styles.js:31]: There is nothing in the figma design about letter spacing, but I found differences in font size between the figma design and the result in development. Am I allowed to change it?

Figma:
зображення

Production:
зображення

Comment (MickhaelSh) [reply src/components/info-card/InfoCard.styles.js:31]: yes, do according to figma

**Review by MickhaelSh — CHANGES_REQUESTED** [review]
do new pr without others` commits

File: src/components/info-card/InfoCard.styles.js (hunk 1 above)
Comment (TSlashDreamy) [reply src/components/info-card/InfoCard.styles.js]: I resized the font-size according to Figma design. The sizes are the same for web, tablet and mobile (24px)

**Comment by TSlashDreamy** [comment]
> do new pr without others` commits

Done it in this branch. Removed other commits and added just mine

**Review by MickhaelSh — APPROVED** [review]

**Pull request merged**